File: src/main/java/ejerciciosJava/Anime.java
package ejerciciosJava;

/**
 * 13. Clase Anime sencilla
 * Crear una clase Anime con los atributos nombre, episodios y género.
 */
public class Anime {
    String nombre;
    int episodios;
    String genero;

    public Anime(String n, int e, String g) {
        this.nombre = n;
        this.episodios = e;
        this.genero = g;
    }

    public String mostrarInfo() {
        return "Nombre: " + nombre + ", Episodios: " + episodios + ", Género: " + genero;
    }
}

File: src/main/kotlin/ejerciciosKotlin/Anime.kt
package org.example.ejerciciosKotlin


/**
 * Crear una clase Anime con los atributos nombre, episodios y género.
 */
class Anime(val nombre:String, val episodios:Int, val genero:String) {

    fun mostrarInfo() {
        println("Nombre: $nombre, Episodios: $episodios, Género: $genero")
    }

}